frontend/src/strings/crawl-workflows/infoText.ts

```diff
@@ -58,6 +58,7 @@ const infoText: Partial<Record<Field, string | TemplateResult>> = {
   ),
   lang: msg(`Websites that observe the browser’s language setting may serve
   content in that language if available.`),
+  proxyId: msg(`Choose a Proxy to crawl through`),
```

Comment:
```suggestion
  proxyId: msg(`Choose a proxy to crawl through`),
```

For consistent casing with other help texts